### ISSUE TITLE
cannot install bourbon using following command ember-cli-bourbon

### ISSUE THREAD

**Issue by VelDeveloper** [ember-cli/ember-cli#7669]
I am using ember-cli=>2.9.1,node=>9.6.1,bower=>1.8.2,npm=>5.6.0

I have created a sample ember app using following command,

**ember new // to create ember project.
ember g template application //To create template with application.hbs.
ember install ember-pikaday // To install ember plugin
ember install ember-cli-sass
ember install ember-cli-bourbon**

The above ember-cli-bourbon command fails with following exception,
---
```
Cannot find module 'resolve'
Error: Cannot find module 'resolve'
   at Function.Module._resolveFilename (module.js:557:15)
   at Function.Module._load (module.js:484:25)
   at Module.require (module.js:606:17)
   at require (internal/module.js:11:18)
   at Object.<anonymous> (/Users/vadivel/POC/ember-
    project/sampleProject/node_modules/ember-cli-bourbon/index.js:5:21)
   at Module._compile (module.js:662:30)
   at Object.Module._extensions..js (module.js:673:10)
   at Module.load (module.js:575:32)
   at tryModuleLoad (module.js:515:12)
   at Function.Module._load (module.js:507:3)
```
```
I have also tried following way,

npm install bourbon //successfully installed
ember install ember-cli-install // Fails with same error as described above.
Any help or hint would be appreciated.
```

**Comment by rwjblue** [comment]
This is a bug in Ember-cli-bourbon, I reported it over there for you:

https://github.com/yapplabs/ember-cli-bourbon/issues/23

**Comment by lukemelia** [comment]
"Resolved" in ember-cli-bourbon 2.0.1
